cloudify_aws/eks/resources/node_group.py

```diff
@@ -44,16 +44,44 @@ class EKSNodeGroup(EKSBase):
     def __init__(self, ctx_node, resource_id=None, client=None, logger=None):
         EKSBase.__init__(self, ctx_node, resource_id, client, logger)
         self.type_name = RESOURCE_TYPE
-        self.describe_param = {}
+        self._describe_param = {}
+
+    @property
+    def describe_params(self):
+        if not self._describe_param:
+            cfg = self.ctx_node.properties['resource_config']
+            cluster_name = cfg.get(
+                CLUSTER_NAME) or cfg['kwargs'].get(CLUSTER_NAME)
+            node_group_name = cfg.get(
+                NODEGROUP_NAME) or cfg['kwargs'].get(NODEGROUP_NAME)
+            self._describe_param = {
+                CLUSTER_NAME: cluster_name,
+                NODEGROUP_NAME: node_group_name
+            }
+        return self._describe_param
+
+    @describe_params.setter
+    def describe_params(self, value):
+        self._describe_param = value
+
+    def wait_for_status(self):
+        self.logger.info('Performing wait for node group.')
+        try:
+            self.wait_for_nodegroup(
+                self.describe_params, 'nodegroup_active',
+                max_attempt=30)
+        except WaiterError:
+            raise OperationRetry('Waiting for nodegroup...')
```

Comment:
Maybe we should also add this to cluster.py since we know it uses the same thing?